index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+    </style>
+    <title>Prototype Mobile Portafolio</title>
+</head>
+<body>
+    <main class="container">
+        <header>
+            <h3>WELCOME</h3>
+            <section class="nav-bar">
+                <div class="line"><h3></h3></div>
+                <div class="line"><h3></h3></div>
+                <div class="line"><h3></h3></div>
+            </section>
```

Comment:
Kindly wrap this part with `nav` tag for semantic purposes, so it will be like:
```
<header>
   <nav>
       your code .......
   </nav>
</header>
```